### ISSUE TITLE
Icon for an Arcade cabinet / Bartop

### ISSUE THREAD

**Issue by MathijsG** [Templarian/MaterialDesign#7362]
### Before submitting a new icon request, please confirm the following:

- [X] I have [searched all issues](https://github.com/Templarian/MaterialDesign/issues?q=is%3Aissue) to make sure there isn't a request for this icon.
- [X] I have [searched the current library](https://pictogrammers.com/library/mdi) to make sure the icon doesn't already exist.
- [X] I understand that [brand icons are no longer being added](https://pictogrammers.com/docs/library/mdi/guides/brand-icons/) and I can use [SimpleIcons](https://simpleicons.org/) for my brand icon needs.
- [X] I only intend to request a single icon (or a few near-identical ones) in this issue.

### What would you use this icon for?

For my arcade cabinet. I think a lot of thinkerers and potential Home Assistant users might own a bartop or a real arcade cabinet these days. Unfortunately the MDI library mostly consists of controller icons, but no actual arcade cabinet icon.

### Give us an example of what this icon might look like.

Something like this:

![image](https://github.com/Templarian/MaterialDesign/assets/529552/d6bd08a1-d8c2-4110-b76a-107604dbf5b6)

![image](https://github.com/Templarian/MaterialDesign/assets/529552/15c8278c-f1e8-41ac-ba20-140c9569bfc9)

**Comment by Dragilooping** [comment]
![Image](https://github.com/user-attachments/assets/30656f33-a4f5-4adb-90df-93c17a1e8100)
[arcade.zip](https://github.com/user-attachments/files/18707036/arcade.zip)

**Comment by MathijsG** [comment]
To be honest because of the frog perspective angle I have a bit of a hard time seeing this as an arcade machine, only after looking for a while I get this. I think the main purpose of an icon is for people to capture it more immediate. Would it be an option to do the angle more from above, and maybe a bit sideways as well?

**Comment by Dragilooping** [comment]
Yes you're right, not a great choice. Is it better this way ?

![Image](https://github.com/user-attachments/assets/1422fc75-6050-4c04-bd50-610daf88b830)
[arcade.zip](https://github.com/user-attachments/files/18707553/arcade.zip)

**Comment by MathijsG** [comment]
Yeah that looks a lot more like it! Although I would make the top bar much smaller as it never is as deep as the cabinet itself.

**Comment by Dragilooping** [comment]
ok, should be good now :

![Image](https://github.com/user-attachments/assets/944353eb-4789-4270-90a5-360611c935ef)
[arcade.zip](https://github.com/user-attachments/files/18707925/arcade.zip)

**Comment by Dragilooping** [comment]
I had to remake it. Please tell me what you think of this :

![Image](https://github.com/user-attachments/assets/ccd28924-9af8-460e-80a2-5a2a2b663857)
[arcade.zip](https://github.com/user-attachments/files/19322191/arcade.zip)

**Comment by MathijsG** [comment]
To me it doesn't really capture the essence, like the referenced logo's do.

What's the major thing about an arcade. It's about the game on the screen, and the controls. Both are abstracted away in your designs, the controls, which are so iconic in recognizing an arcade are just 2 pixels.

I would need to look twice or thrice to spot that this would be an arcade.

**Comment by MathijsG** [comment]
Something I made with the new ChatGPT image creator:

![Image](https://github.com/user-attachments/assets/df3de58f-79db-4924-95b8-0702fd2c93c4)

Would something like this work?

**Comment by Dragilooping** [comment]
It is pretty good ! It will need some work to match the MD guidelines
